drivers/builtin/src/entropy_poll.h

```diff
@@ -18,19 +18,22 @@
 extern "C" {
 #endif
 
-#if defined(MBEDTLS_PLATFORM_GET_ENTROPY_ALT)
+#define MBEDTLS_ENTROPY_POLL_PLATFORM_MIN        32
 
-#define MBEDTLS_ENTROPY_MIN_HARDWARE        32
-
-int mbedtls_hardware_poll(void *data, unsigned char *output, size_t len, size_t *olen);
-
-#else /* MBEDTLS_PLATFORM_GET_ENTROPY_ALT */
-
-#define MBEDTLS_ENTROPY_MIN_PLATFORM        32
-
-int mbedtls_platform_entropy_poll(void *data, unsigned char *output, size_t len, size_t *olen);
-
-#endif /* MBEDTLS_PLATFORM_GET_ENTROPY_ALT */
+/**
+ * This function get called from the entropy module when it's gathering entropy
+ * data. Backends are:
+ * - on Windows: no need to define MBEDTLS_PLATFORM_GET_ENTROPY_ALT and
+ *              CryptoAPI will be used.
+ * - on Linux: no need to define MBEDTLS_PLATFORM_GET_ENTROPY_ALT and
+ *              /dev/urandom will be used.
```

Comment:
As in the public documentation: not just `/dev/urandom`, but also `getrandom` and variants are preferred. And this is not just on Linux but also some Linux-like systems and most BSD variants.